rasa_nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -136,7 +136,7 @@ def __init__(self,
         self._check_tensorflow()
         super(EmbeddingIntentClassifier, self).__init__(component_config)
 
-        self._load_params()
+        self._load_params(**component_config)
```

Comment:
I think this should rather be `self.component_config` and we should remove the deep copy & overwrite implementation in `_load_params`. This behaviour is already implemented in the init method of `Component`.